src/olympia/reviewers/utils.py

```diff
@@ -1305,6 +1326,11 @@ def approve_multiple_versions(self):
     def block_multiple_versions(self):
         raise NotImplementedError  # only implemented for unlisted below.
 
+    def clear_needs_human_review(self):
+        """clears human review all versions in this channel."""
+        self.clear_all_needs_human_review_flags_in_channel(mad_too=False)
```

Comment:
Should a test be added that calling `clear_needs_human_review` does not clear the flag for MAD? I see a code change around this, but no test for it.

Reply:
https://github.com/mozilla/addons-server/pull/20386/commits/5379146e5c70d0be4759578115f308e913e0d22b